scripts/bump.ts

```diff
@@ -0,0 +1,67 @@
+import * as fs from 'fs';
+import simpleGit, { SimpleGitOptions } from 'simple-git';
+
+// Increments the specified version based on the release type
+function incrementVersion(version: string, release: string): string {
+  const parts = version.split('.').map((part) => parseInt(part, 10));
+
+  if (release[0] === 'major') {
+    parts[0]++;
+    parts[1] = 0;
+    parts[2] = 0;
+  } else if (release[0] === 'minor') {
+    parts[1]++;
+    parts[2] = 0;
+  } else if (release[0] === 'patch') {
+    parts[2]++;
+  } else {
+    parts[2]++;
+  }
+
+  return parts.join('.');
+}
+
+// Reads the current version from the package.json file
+function readPackageJson(): any {
+  const packageJson = fs.readFileSync('package.json', { encoding: 'utf-8' });
+  return JSON.parse(packageJson);
+}
+
+// Writes the new version to the package.json file
+function writePackageJson(packageJson: any): void {
+  fs.writeFileSync('package.json', JSON.stringify(packageJson, null, 2), {
```

Comment:
I'm worried this will clobber any formatting in the JSON file. Have you tested for that?